### ISSUE TITLE
`-Wc` does not pass flag to `wcc`

### ISSUE THREAD

**Issue by jayschwa** [open-watcom/open-watcom-v2#541]
`owcc -?` lists "-Wc,<option> pass any option to WCC". I cannot get it to work. Below is a snippet of a build.
```
owcc -bDOS -march=i286 -fpmath=287 -Isrc -Isrc/cat3d -mcmodel=l -Wc,-zt -v   -c -o src/wiz.o src/wiz.c
Open Watcom C/C++ x86 32-bit Compiler Driver Program
Version 2.0 beta Jun 16 2019 15:54:54 (64-bit)
[...]
	wcc -bt=dos -fp2 -Isrc -Isrc/cat3d -ml -2 -fo=src/wiz.o -fr src/wiz.c
```
`owcc` is given `-Wc,-zt`, but `-zt` does not show up in the `wcc` command.

**Comment by jayschwa** [comment]
I have realized that I can use `-ffar-data-threshold` to do what I was trying to do with `-Wc,-zt`. However, the original issue still seems like something that needs to be fixed.

**Comment by jmalak** [comment]
Thanks for your bug report.
I confirm it as bug.
I will fix it ASAP.

**Comment by jmalak** [comment]
it is fixed by a7fceb783eb2f49c2b8d543d96e2830247615de7 commit.
now translated owcc command line is
wcc -bt=dos -fp2 -Isrc -Isrc\cat3d -ml -zt -2 -fo=src\wiz.o -fr src\wiz.c
